docker-build/README.md

```diff
@@ -1,28 +1,30 @@
-| `gar_name`            | Yes      | Name of the GAR repository                                                                                                                                                                |
-| `project_id`          | Yes      | GCP project ID                                                                                                                                                                            |
-| `image_build_context` | Yes      | Path to the Docker build context (default: `./`)                                                                                                                                          |
-| `image_tag_metadata`  | No       | Metadata to append to the image tag.<br><br>For example, for a workflow triggred by a git tag `v1.2.3` and an `image_tag_metadata` value `dev`, the final image tag will be `v1.2.3--dev` |
+| `gar_name`            | Yes      | Name of the GAR repository (typically `<tenant name>-prod`)                                                                                                                               |
+| `project_id`          | Yes      | GCP project ID (typically `moz-fx-<tenant name>-prod`)                                                                                                                                    |
+| `image_build_context` | No       | Path to the Docker build context (default: `./`)                                                                                                                                          |
+| `gar_location`        | No       | GCP region where GAR is located (default: `us`)                                                                                                                                           |
 | `should_tag_ghcr`     | No       | Whether or not to generate image tags for Github Container Registry (default: `false`)                                                                                                    |
 | `should_tag_latest`   | No       | Whether or not to tag the image(s) as `latest` (default: `false`)                                                                                                                         |
+| `image_tag_metadata`  | No       | Metadata to append to the image tag.<br><br>For example, for a workflow triggred by a git tag `v1.2.3` and an `image_tag_metadata` value `dev`, the final image tag will be `v1.2.3--dev` |
 
 ## Outputs
 
-| Name         | Description                                  |
-| ------------ | -------------------------------------------- |
-| `image_tags` | The list of generated Docker images and tags |
+| Name         | Description                                             |
+| ------------ | ------------------------------------------------------- |
+| `image_tags` | A newline-delimited list of generated Docker image tags |
 
 ## Example
 
+### Usage
+
```

Comment:
This example should include text that describes what it does and the use case for it. Something like:

> For example, I want to build an image for my service and include GHCR data. My service uses image tag metadata to indicate the something something.